### ISSUE TITLE
How to show oneShot effect in DialogFragment's View?

### ISSUE THREAD

**Issue by huuchi207** [plattysoft/Leonids#80]
i use this code in DialogFragment: 
```
ParticleSystem ps = new ParticleSystem(getActivity(), 70, R.drawable.star_pink, 800);
                                ps.setScaleRange(0.7f, 1.3f);
                                ps.setSpeedRange(0.1f, 0.25f);
                                ps.setRotationSpeedRange(90, 180);
                                ps.setFadeOut(200, new AccelerateInterpolator());
                                ps.oneShot(imageView, 70);
```
but the effect shows in Activity(behind Dialog) :(

**Comment by plattysoft** [comment]
By default, the  ParticleSystem creates a full screen transparent view that is attached to the content view of the Activity as a first child - that is, on the very background -

To change that behaviour, you can provide another View as the parent, so the particle system can have views behind it. have you tried that?

**Comment by huuchi207** [comment]
"you can provide another View as the parent". How can I try that? Can u give me an example code?

**Comment by huuchi207** [comment]
I can't find out the constructor as you mention in documentation in your lib:
" And another constructor that receives a parent viewgroup and drawable for use in places where it is not practical to pass a reference to an Activity

ParticleSystem(ViewGroup parentView, int maxParticles, Drawable drawable, long timeToLive)"

**Comment by plattysoft** [comment]
yes, that is the constructor you want to use. Yes, it is a ViewGroup, not a View, sorry I was talking from memory.

**Comment by huuchi207** [comment]
![untitled](https://user-images.githubusercontent.com/25658269/28516872-d775e42e-708c-11e7-81ad-6ee637aec9f1.png)
here is particle system 's contructors list. None of them has ViewGroup argument :(

**Comment by plattysoft** [comment]
Are you sure you are using the latest published version? It is one of the release notes https://github.com/plattysoft/Leonids/releases/tag/1.3.2

**Comment by plattysoft** [comment]
and you can see it in the code of that particular tag: https://github.com/plattysoft/Leonids/blob/1.3.2/LeonidsLib/src/main/java/com/plattysoft/leonids/ParticleSystem.java#L88

**Comment by huuchi207** [comment]
ok let me try your suggest. Please change your construction 's setup section. It 's still guiding version 1.3.1 :D

**Comment by huuchi207** [comment]
i solved my problem by using version 1.3.2. Thank you for your lib and your solution!
